api/v1/helpers.go

```diff
@@ -51,6 +51,7 @@ const (
 
 	VerticaDBNameKey = "verticaDBName"
 	SandboxNameKey   = "sandboxName"
+	invalidNameChars = "$=<>`" + `'^\".@*?#&/:;{}()[] \~!%+|,`
```

Comment:
Not sure whether this covers all the invalid characters or not, but would it be better to use a whitelist to check the valid characters as described on this page?
https://docs.vertica.com/latest/en/sql-reference/language-elements/identifiers/

Reply:
this is the same validation vclusterops does, so I think it is fine